### PR TITLE
XW-1368 | remove API from Hapi

### PR DESCRIPTION
**Pull request by bognix**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1368
* http://wikia-inc.atlassian.net/browse/XW-1377
* https://github.com/Wikia/mercury/pull/2363/files

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM

**Comment by hakubo** [comment]
we should remove tests as well

File: server/config/localSettings.base.js [hunk 1]
```diff
@@ -158,7 +158,6 @@ import deepExtend from 'deep-extend';
  */
 
 const localSettings = {
-	apiBase: '/api/mercury',
```
Comment (vforge) [inline server/config/localSettings.base.js]: Please remove that also from JSDoc above - https://github.com/Wikia/mercury/blob/XW-1368/server/config/localSettings.base.js#L51

**Comment by vforge** [comment]
lgtm

**Comment by rogatty** [comment]
LGTM

**Pull request merged**